src/assets/netteForms.compat.js

```diff
@@ -0,0 +1,10 @@
+(function(global) {
+
+	if (typeof define === 'function' && define.amd || typeof module === 'object' && typeof module.exports === 'object' || !global || !global.Nette) {
```

Comment:
The netteForms.compat.js script is supposed to provide backwards compatibility with the old behaviour of the netteForms.js script - which called global.Nette.initOnLoad() only if loaded in a browser environment, not when defining an AMD module or when required as a Node.js module. This condition is to make sure the behaviour stays the same, although admittedly _not_ including the netteForms.compat.js script would achieve the same behaviour in AMD or Node.js environments.